### DISCUSSION TITLE
SoP: technical or story telling ?

### DISCUSSION THREAD

**Discussion by anhtienng** [nguyenthanhvuh/phd-cs-us#14] [General]
Hello everyone.

I saw various SoP samples (for CS and top universities) that focus on explaining projects in details (technical contribution, role in projects, results). I have 2 following questions:
- If SOP is written like that, is it a detailed (and duplicated) version of the CV ?
- There are suggestions that SoP should be like story telling to emphasize on research motivations (instead of being technical like above). Is this a good suggestion ?

**Comment by nguyenthanhvuh** [comment]
either way is fine,  pick the style you're most comfortable with and write it.

**Comment by anhtienng** [comment]
thank you

**Comment by anhtienng** [comment]
Is this good to use technical words of a small research area (e.g ML in medical images) in a SOP @nguyenthanhvuh ? 
I am afraid that some of the admission committees are not familiar with these terminologies.

**Reply by nguyenthanhvuh** [reply]
I think some small amount of technical jargons is fine.  As long as the overall message is still clear.

**Reply by anhtienng** [reply]
thank you
